cypress/support/index.ts

```diff
@@ -24,7 +24,9 @@ import "cypress-real-events/support";
 require('cypress-xpath');
 
 beforeEach(() => {
-  Cypress.Cookies.preserveOnce('JSESSIONID', 'BMS_TOK')
+ Cypress.Cookies.defaults({
+    preserve: ['JSESSIONID', 'BMS_TOK'],
+  })
```

Comment:
I think the difference is just Cookies.defaults() is a global setting of default cookies. So we can add it directly to index.ts instead to the beforeEach hook. I just used this to avoid the known issue for Cookies.preserveOnce() mentioned by Aldrin in the ticket - https://github.com/cypress-io/cypress/issues/15166. 
BMS throws 2 JSESSIONID with 2 diff path (ibpworkbench and fieldbook) when user navigates to the Manage Studies, thus using preserveOnce will delete the cookies
![image](https://user-images.githubusercontent.com/7791554/143970992-dbff27b9-f1f0-4f47-b9a5-338133d149d5.png)